I'm getting the error:

```
##[error].packages\microsoft.dotnet.genfacades\5.0.0-beta.19577.4\build\Microsoft.DotNet.GenPartialFacadeSource.targets(30,5): error : Did not find type 'System.Runtime.CompilerServices.SkipLocalsInitAttribute' in any of the seed assemblies.
```

What's a seed assembly?

--- a/src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/SkipLocalsInitAttribute.cs
+++ b/src/libraries/System.Private.CoreLib/src/System/Runtime/CompilerServices/SkipLocalsInitAttribute.cs
@@ -6,17 +6,17 @@ namespace System.Runtime.CompilerServices
 {
     /// <summary>
     /// Used to indicate to the compiler that the <c>.locals init</c> flag should
-    /// not be set in nested method headers when emitting to metadata. This
-    /// attribute is unsafe because it may reveal uninitialized memory to the
-    /// application in certain instances (e.g., reading from uninitialized
+    /// not be set in nested method headers when emitting to metadata.
+    /// </summary>
+    /// <remarks>
+    /// This attribute is unsafe because it may reveal uninitialized memory to
+    /// the application in certain instances (e.g., reading from uninitialized
     /// stackalloc'd memory). If applied to a method directly, the attribute
     /// applies to that method and all nested functions (lambdas, local
     /// functions) below it. If applied to a type or module, it applies to all
-    /// methods nested inside.
-    /// </summary>
-    /// <remarks>
-    /// This attribute is intentionally not permitted on assemblies. Use at the
-    /// module level instead to apply to multiple type declarations.
+    /// methods nested inside. This attribute is intentionally not permitted on
+    /// assemblies. Use at the module level instead to apply to multiple type
+    /// declarations.
     /// </remarks>
     [AttributeUsage(AttributeTargets.Module
         | AttributeTargets.Class

